### ISSUE TITLE
Protect From Hostile Spawn does not work

### ISSUE THREAD

**Issue by Dr-WeiAL** [thexaero/open-parties-and-claims#505]
When I turn this option on, monsters still spawn naturally in the block, only spawn eggs and summon commands fail to spawn

**Comment by thexaero** [comment]
Weird, that's the opposite of how it should work.

**Comment by Dr-WeiAL** [comment]
Indeed, I feel the same way.

**Comment by thexaero** [comment]
So what MC, mod loader and mod version are you using?

**Comment by Dr-WeiAL** [comment]
The version I use is as follows
Minecraft version 1.20.1
Forge version 47.2.20
Mod version 0.23.0

**Comment by thexaero** [comment]
Thanks!

**Comment by thexaero** [comment]
Still haven't been able to reproduce this and it's probably too late to ask what other mods you were using.

**Comment by Dr-WeiAL** [comment]
Uh, indeed, the current situation is that no monsters can be spawned by any means, so maybe it's kind of fixed?
Is it a bit much to not even be able to spawn monsters with spawn eggs and commands though?
![Image](https://github.com/user-attachments/assets/bae6cf33-9a91-4f90-8d89-9b9d388c16e1)

**Comment by thexaero** [comment]
Spawn eggs and commands work completely fine for me. Do they start working for you if you disable hostile mob spawn protection?

**Comment by Dr-WeiAL** [comment]
`Protect From Hostile Spaawn : OFF`
That is, the default settings

`Protect From Hostile Spaawn : ON`
In this case, monsters cannot be spawned in any way, whether by natural generation, spawn eggs, or the command

It's probably too much to put into words, this gif should explain everything
![Image](https://github.com/user-attachments/assets/2e893524-2b23-423e-9cac-40f5c7f20e46)


Also I still get this problem with just these mods
```
Xaeros_Minimap_25.2.6_Forge_1.20
XaerosWorldMap_1.39.9_Forge_1.20
open-parties-and-claims-forge-1.20.1-0.24.0
```

**Comment by thexaero** [comment]
If you can figure out which mod causes this, it'd be very helpful. Otherwise I can't do anything. That's not how OPAC normally works.

**Comment by Dr-WeiAL** [comment]
Uh... what that last paragraph above means is that I only installed those three mods and nothing else, even the config was loaded for the first time.

[latest.log](https://github.com/user-attachments/files/20430772/latest.log)

**Comment by thexaero** [comment]
Then you probably just need to update Forge to a newer version that fixed this issue.

**Comment by Dr-WeiAL** [comment]
The Forge 47.4.0 currently in use is the latest version, and even if you switch to the latest version of NeoForge 47.1.105 you'll have the same problem.

Also I tested it again and the problem exists in version 1.21.1 as well

**Comment by thexaero** [comment]
Something in your case makes these spawn events appear as "natural". I have no idea how that is possible.

**Comment by thexaero** [comment]
Ok, now I am able to reproduce it for some reason. Because of course lol.
Created a new issue for it: #678

**Comment by Dr-WeiAL** [comment]
Uh... I tested Fabric and it did work fine, so all along you were using Fabric to test for problems?

**Comment by thexaero** [comment]
No, I was using Forge. I think I have a false memory of testing the spawn egg and command thing on 1.20.1. Probably tested everything on an older MC version first, where everything worked, then tested just the spawn protection on 1.20.1, saw that it worked and concluded that I didn't have the issue, since that was the main complaint. Looks like it was 2 unrelated issues rather than 1.

**Comment by Dr-WeiAL** [comment]
Well, it's good to find out what the problem is.
